unlock-app/src/reducers/providerReducer.js

```diff
@@ -1,10 +1,15 @@
 import { SET_PROVIDER } from '../actions/provider'
 import configure from '../config'
 
-const config = configure(global)
+const isServer = typeof window === 'undefined'
```

Comment:
Same here ;) Moving that logic to `configure` would make sure we do not forget to add it in all files ;)

Reply:
Done